crates/pipeline_manager/src/api/mod.rs

```diff
@@ -416,6 +416,10 @@ pub async fn run(db: Arc<Mutex<ProjectDB>>, api_config: ApiServerConfig) -> AnyR
                         api_config.dev_mode,
                         actix_cors::Cors::permissive(),
                     ))
+                    .wrap(Condition::new(
+                        !api_config.dev_mode,
+                        actix_cors::Cors::default().allowed_origin("https://www.feldera.com"),
```

Comment:
you might want this configurable through a config/cmdline instead of a hard-coded string ... can imagine scenarios where 3rd party user wants their own Cors list